lib/ddtrace/contrib/cucumber/integration.rb

```diff
@@ -0,0 +1,40 @@
+require 'ddtrace/contrib/integration'
+require 'ddtrace/contrib/cucumber/configuration/settings'
+require 'ddtrace/contrib/cucumber/patcher'
+require 'ddtrace/contrib/integration'
+
+module Datadog
+  module Contrib
+    module Cucumber
+      # Description of Cucumber integration
+      class Integration
+        include Contrib::Integration
+
+        MINIMUM_VERSION = Gem::Version.new('4.0.0')
```

Comment:
Something I think worth understand is what min version is reasonable here?

Looking at some Gem Stats it seems like 4.0 only was released a few months ago (June 2020), and 3.0 has much broader usage, more than 4+5 combined.

https://rubygems.org/gems/cucumber/versions

Is there a reason we've chosen to use 4.0 as the min version here? If there's a good reason here, let's document that or understand why.

Reply:
I think it should be compatible with version 3 too.

Reply:
The small issue with cucumber 3 is that the step hooks are triggered for before/after step hooks too, but it's not critical.